recording/lib/reporter.ex

```diff
@@ -30,11 +38,22 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Reporter do
   end
 
   @spec add_track(pid(), Track.t(), filename(), pos_integer()) :: :ok
-  def add_track(reporter, track, filename, offset) do
-    track = Map.put(track, :offset, offset)
+  def add_track(reporter, track, filename, timestamp) do
+    track = Map.put(track, :offset, timestamp)
+
     GenServer.cast(reporter, {:add_track, track, filename})
   end
 
+  @spec start_track(pid(), Track.id(), pos_integer()) :: :ok
+  def start_track(reporter, track_id, timestamp) do
```

Comment:
in `end_track` there is `end_timestamp` I suggest `start_timestamp` instead of `timestamp` for consistency.